packages/rx_bloc_cli/mason_templates/bricks/lib_auth/__brick__/lib/lib_auth/services/user_account_service.dart

```diff
@@ -38,6 +43,26 @@ class UserAccountService {
       password: password,
     );
 
+    await _handleToken(authToken);
+  }
+{{#enable_social_login}}
+  /// Authenticates user using social media service
+  ///
+  /// After successful login saves the auth `token` and `refresh token` to
+  /// persistent storage and loads the user permissions.
+  Future<void> socialLogin() async {
+    if (_socialLoginService == null) {
```

Comment:
I believe the social login must be not nullable in case enable_social_login is true

Reply:
You are right. I added thid to have propper message if we miss to implement it while adding a new social login option. There is no other benefit of it, I think.

Reply:
enable_social_login might be set to false. However, I can delete this check.